docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -421,14 +420,14 @@ helm install $HELM_RELEASE_NAME camunda/camunda-platform \
 
 #### Verification
 
-The following command will show the deployed pods of the newly created region.
+The following command will show the pods deployed in the newly created region.
 
-Depending on your chosen `clusterSize`, you should see that half of the amount are spawned in Zeebe brokers.
+Depending on your chosen `clusterSize`, you should see that half of the amount is spawned in Zeebe brokers.
```

Comment:
```suggestion
Half of the amount of your set `clusterSize` is used to spawn Zeebe brokers.
```
^ This could potentially be included with the example line below. I'm unsure with how it's currently written if this suggestion is what's meant, so shortening it up to be clear may help.